domains/miskode.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "miskode",
+        "email": "miskonx@gmail.com"
+    },
+
+    "record": {
+        "CNAME": "miskode.github.io"
```

Comment:
404 page

Reply:
It is no longer a 404, content has been uploaded.